docs/generate-devsite-help.sh

```diff
@@ -36,24 +36,24 @@ cd output/devsite-help
 
 # Create the docs metadata. We assume we may need to refer to any of the utility libraries.
 # We also refer to Datastore, PubSub and Storage.
-python -m docuploader create-metadata \
+dotnet run --project ../../../tools/Google.Cloud.Tools.DocUploader -- create-metadata \
```

Comment:
You need these changes in generate-devsite-utilities.sh too.

(And as elsewhere, I think it would be good to use the repo root as an anchor, although at least this is relative.)

Reply:
done

Reply:
I'm not seeing that file in the PR... missed a "git add" perhaps?

Reply:
oh I think I missed that, let me do that.

Reply:
updated it please have a look.